server/__init__.py

```diff
@@ -248,7 +256,9 @@ def create_app():
   app.config['BABEL_TRANSLATION_DIRECTORIES'] = 'i18n'
 
   # load disaster dashboard data from GCS
-  if os.environ.get('FLASK_ENV') in ['autopush', 'local', 'dev']:
+  if os.environ.get('FLASK_ENV') in [
+      'autopush', 'local', 'dev', 'local-stanford', 'stanford'
+  ]:
```

Comment:
we should also move this into the "register_stanford_routes" function (though this is mostly temporary... so mostly a note for future that we want the flask app to only contain the smallest amount of required resources for the pages it serves)

Reply:
Done!